tgui/packages/tgui/interfaces/Vote.js

```diff
@@ -0,0 +1,229 @@
+                    onClick={() => act("toggle_gamemode")} >
+                    Disable{!avm ? "d" : ""}
+                  </Button.Checkbox>
+                )}
+              </Box>
+            </Flex.Item>
+            <Flex.Item>
+              <Button disabled={!upper_admin} onClick={() => act("custom")}>
+                Create Custom Vote
+              </Button>
+            </Flex.Item>
+          </Flex>
+        </Collapsible>
+        <Collapsible title="View Voters">
+          <Box mt={2} width="100%" height={6} overflowY="scroll">
+            {voting.map(voter => {
+              return <Box key={voter}>{voter}</Box>;
+            })}
+          </Box>
+        </Collapsible>
+      </Section>
+    </Flex.Item>
+  );
+};
+
+// Display choices as buttons
+const ChoicesPanel = (props, context) => {
+  const { data } = useBackend(context);
+  const { mode, choices } = data;
+
+  let content;
+  if (choices.length === 0) {
+    content = "No choices available!";
+  }
+  // Single box for most normal vote types
+  else if ((choices.length < 10) | (mode === "custom")) {
+    content = (
+      <DisplayChoices
+        choices={choices}
+        tally="Votes:"
+        startIndex={0}
+        margin={1} />
+    );
+  } else {
+    // If there's both too much content, most likely gamemode
+    content = (
+      <Flex justify="space-between" direction="row">
+        <Flex direction="column">
+          <DisplayChoices
+            choices={choices.filter(
+              (choice, index) => index < choices.length / 2
+            )}
+            tally="|"
+            startIndex={0}
+            margin={0} />
+        </Flex>
+        <Flex direction="column" ml={1}>
+          <DisplayChoices
+            choices={choices.filter(
+              (choice, index) => index > choices.length / 2
+            )}
+            tally="|"
+            startIndex={Math.ceil(choices.length / 2)}
+            margin={0} />
+        </Flex>
+      </Flex>
+    );
+  }
+
+  return (
+    <Flex.Item mb={1} grow={1}>
+      <Section fill title="Choices">
+        {content}
+      </Section>
+    </Flex.Item>
+  );
+};
+
+const DisplayChoices = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { selectedChoice } = data;
+
+  return props.choices?.map((choice, i) => (
+    <Flex justify="space-between" direction="row" key={i} mb={props.margin}>
+      <Flex>
+        <Button
+          onClick={() => {
+            act("vote", {
+              index: i + props.startIndex + 1,
+            });
+          }}
+          disabled={
+            choice === props.choices[selectedChoice - props.startIndex - 1]
+          } >
+          {choice.name?.replace(/^\w/, c => c.toUpperCase())}
+        </Button>
+        <Box mt={0.4} ml={1}>
+          {choice === props.choices[selectedChoice - props.startIndex - 1] && (
+            <Icon color="green" name="vote-yea" />
+          )}
+        </Box>
+      </Flex>
+      <Box ml={1}>
```

Comment:
interesting thought if you wanted to freshen up the choices panel: if you split this up a bit you could make the voting options with 0 votes still use the label font